tests/integration/article.test.js

```diff
@@ -133,6 +135,74 @@ describe("# Articles endpoints", () => {
         });
     });
   });
+  describe("Report an article endpoint", () => {
+    it("should be report an article", done => {
+      chai
+        .request(app)
+        .put(`/api/v1/articles/${validSlug}/report`)
+        .set("Authorization", `Bearer ${token}`)
+        .send({ description: "abusive" })
+        .end((err, res) => {
+          expect(res.status).equals(OK);
+          expect(res.body.message).to.contain("Article reported");
+          done();
+        });
+    });
+    it("should should ask for description", done => {
+      chai
+        .request(app)
+        .put(`/api/v1/articles/${validSlug}/report`)
+        .set("Authorization", `Bearer ${token}`)
+        .send({ description: "" })
+        .end((err, res) => {
+          expect(res.status).equals(BAD_REQUEST);
+          expect(res.body.message).to.contain("Please, give a reason");
+          done();
+        });
+    });
+    it("should deny the request if no access-token provided", done => {
+      chai
+        .request(app)
+        .put(`/api/v1/articles/${validSlug}/report`)
+        .end((err, res) => {
+          expect(res.status).equals(UNAUTHORIZED);
+          done();
+        });
+    });
+    it("should report an article", done => {
+      chai
+        .request(app)
+        .put(`/api/v1/articles/${validSlug}/report`)
+        .set("Authorization", `Bearer ${token}`)
+        .send({ description: "abusive" })
+        .end((err, res) => {
+          expect(res.status).equals(OK);
+          expect(res.body.message).to.contain("Article reported");
+          done();
+        });
+    });
+    it("should should ask for description", done => {
```

Comment:
Check to make sure if this case scenario is a duplicate.

Reply:
It was, thanks